tests/admin_scripts/tests.py

```diff
@@ -2206,6 +2206,16 @@ def test_importable_target_name(self):
             "another directory."
         )
 
+    def test_trailing_slash_in_target_app_directory_name(self):
+        app_dir = os.path.join(self.test_dir, 'apps', 'app1')
+        os.makedirs(app_dir)
+        _, err = self.run_django_admin(['startapp', 'app', os.path.join('apps', 'app1', '')])
+        self.assertNoOutput(err)
+        self.assertEquals(
+            ['migrations', 'models.py', '__init__.py', 'apps.py', 'admin.py', 'tests.py', 'views.py'],
+            os.listdir(app_dir)
+        )
```

Comment:
I think it's enough to check `apps.py`. Also, `assertEquals()` is a deprecated alias of `assertEqual()`.